internal/pkg/cli/pipeline_init.go

```diff
@@ -89,6 +100,7 @@ type initPipelineVars struct {
 	repoURL           string
 	repoBranch        string
 	githubAccessToken string
+	typ               string
```

Comment:
Can we rename this to something like `pipelineType`? Because it looks incomplete as soon as we look at it.